File: cpp_e9zMZ/cpp/08-mang/119.cpp
#include <iostream>
using namespace std;

int main() {
    int score[] = {2, 9, 8, 4, 10};
    string student[] = {"Hiếu", "Dũng", "Minh", "Cường", "Lan"};
    int length = sizeof(score) / sizeof(int);
    int idx = 0;

    for (int i = 0; i < length; i++) {
        int currentScore = score[i];
        idx = currentScore > score[idx] ? i : idx;
    }

    cout << "HV có điểm cao nhất - " << student[idx] << ": " << score[idx];
    return 0;
}

File: python_ToSXs/python/08-mang/119.py
score = [2, 5, 8, 4, 10]
student = ["Han", "Dũng", "Linh", "Cường", "Lan"]

idx = 0
length = len(score)

for i in range(1, length):
    if score[i] > score[idx]:
        idx = i

print(f"HV có điểm cao nhất - {student[idx]}: {score[idx]}")
